eng/cake/dotnet.cake

```diff
@@ -367,48 +372,33 @@ Task("dotnet-diff")
 
 // Tasks for Local Development
 
-Task("VS-DOGFOOD")
```

Comment:
This target doesn't really work. The intention was to open an empty VS and then open an external SLN but you have to do a bit more work to make that work.

Reply:
Could you update the `Development.md` in this PR?

Reply:
Yea, I'm waiting until everyone is happy with this PR and then I'll update `Development.md`